### PR TITLE
Make it run on linux

### PR DESCRIPTION
**Pull request by LeComm**

### CODE REVIEW

File: ogl.c [hunk 1]
```diff
@@ -129,7 +130,8 @@ void ogl_init() {
 }
 
 void ogl_display() {
-	float fog_color[] = {0.5F,0.9098F,1.0F,1.0F};
+	printf("TEST\n");
```
Comment (NotAFile) [inline ogl.c:133]: whoops

**Review by NotAFile — CHANGES_REQUESTED** [review]

File: ogl.c (hunk 1 above)
Comment (LeComm) [reply ogl.c:133]: Not going to pretend like I still remember what I did there, but after my experiences with the renderer, that one could totally have been on purpose hehe

**Comment by iamgreaser** [comment]
BetterSpades at the moment currently works on Linux just fine without this PR. Should this PR be closed?

**Comment by xtreme8000** [comment]
@iamgreaser this PR is for a different branch

**Pull request closed without merging**